.github/workflows/examples-infineon.yaml

```diff
@@ -59,15 +59,23 @@ jobs:
               timeout-minutes: 10
               run: |
                 scripts/run_in_build_env.sh \
-                  "scripts/build/build_examples.py --no-log-timestamps --target-glob 'infineon-*' build"
+                  "scripts/build/build_examples.py --no-log-timestamps --target-glob 'infineon-p6-lock' build"
```

Comment:
`--target` should be used instead of glob if no globbing is being done

Reply:
Any reason we separate this out into 2 separate build steps instead of 'build all in one' ?

Reply:
@andy31415  : Thanks for the review: I have removed glob from the build. 
Yes.. wanted to separate two example app build for easy debug in case of failures.